assets/cubyz/blocks/glass/indigo.zig.zon

```diff
@@ -6,7 +6,7 @@
 	},
 	.transparent = true,
 	.hasBackFace = true,
-	.absorbedLight = 0xd0c16d,
+	.absorbedLight = 0x2f3e92,
```

Comment:
I don't think this is correct, this makes it absorb more blue light, turning it yellow.